### PR TITLE
(1866) Rename Transfer to OutgoingTransfer

### PR DESCRIPTION
**Pull request by pezholio**
As part of the [work to record incoming reciept of a transfer](https://trello.com/c/mIQGZPwj/1822-the-application-must-store-the-outgoing-and-incoming-transfers), we need to rename the existing `Transfer` model to `OutgoingTransfer`. To make reviewing easier, I've decided to submit this as a seperate PR first.

### CODE REVIEW

**Comment by pezholio** [comment]
[Rename Transfer to OutgoingTransfer](https://trello.com/c/QUK1VJ6Q/1866-rename-transfer-to-outgoingtransfer)

File: app/controllers/staff/activity_financials_controller.rb [hunk 1]
```diff
@@ -16,6 +16,6 @@ def show
     @budget_presenters = @budgets.includes(:parent_activity, :providing_organisation).map { |budget| BudgetPresenter.new(budget) }
     @planned_disbursement_presenters = @planned_disbursements.map { |planned_disbursement| PlannedDisbursementPresenter.new(planned_disbursement) }
     @implementing_organisation_presenters = activity.implementing_organisations.map { |implementing_organisation| ImplementingOrganisationPresenter.new(implementing_organisation) }
-    @transfers = policy_scope(activity.source_transfers).map { |transfer| TransferPresenter.new(transfer) }
+    @transfers = policy_scope(activity.source_transfers).map { |transfer| OutgoingTransferPresenter.new(transfer) }
```
Comment (CristinaRO) [inline app/controllers/staff/activity_financials_controller.rb:19]: This is up to you, but would it help or hinder to rename the instance variables(s) too?

**Review by CristinaRO — APPROVED** [review]
One question, not blocking.

And, of course, conflicts in the changelog. /o\

File: app/controllers/staff/activity_financials_controller.rb (hunk 1 above)
Comment (pezholio) [reply app/controllers/staff/activity_financials_controller.rb:19]: I did consider it, but there's going to be bunch of work coming down the pipe with concerns etc, so I didn't think it was worth doing at the time

**Pull request merged**